apps/admin-x-activitypub/src/components/Profile.tsx

```
@@ -326,45 +340,50 @@ const Profile: React.FC<ProfileProps> = ({}) => {
-                            <Heading className='mt-4' level={3}>{userProfile?.name}</Heading>
+                            <Heading className='mt-4' level={3}>{account?.name}</Heading>
                             <span className='mt-1 text-[1.5rem] text-grey-800'>
-                                <span>{userProfile && getUsername(userProfile)}</span>
+                                <span>{account?.handle}</span>
                             </span>
-                            {(userProfile?.summary || attachments.length > 0) && (
+                            {(account?.bio || customFields.length > 0) && (
                                 <div ref={contentRef} className={`ap-profile-content transition-max-height relative text-[1.5rem] duration-300 ease-in-out [&>p]:mb-3 ${isExpanded ? 'max-h-none pb-7' : 'max-h-[160px] overflow-hidden'} relative`}>
                                     <div
-                                        dangerouslySetInnerHTML={{__html: userProfile?.summary ?? ''}}
+                                        dangerouslySetInnerHTML={{__html: account?.bio ?? ''}}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)